frontend/tsconfig.app.json

```diff
@@ -3,7 +3,7 @@
         "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.app.tsbuildinfo",
         "target": "ES2020",
         "useDefineForClassFields": true,
-        "lib": ["ES2020", "DOM", "DOM.Iterable"],
+        "lib": ["ES2023", "DOM", "DOM.Iterable"],
```

Comment:
why the bump to es2023? will the Nexus 7 in the hacklab support it?

Reply:
the target is still ES2020

Reply:
> why the bump to es2023? will the Nexus 7 in the hacklab support it?

Array.findLast needs es2023

Reply:
The findLast thing was to address this bug:
![Screenshot 2024-11-21 162358](https://github.com/user-attachments/assets/94dcf173-7a73-4599-9ee9-b16fdc72309c)
But I think more needs to be done to address the bug since it doesn't really explain why the bug occurs with only 1 currentEvent.